components/camel-spring/src/main/java/org/apache/camel/spring/SpringCamelContext.java

```diff
@@ -301,4 +301,36 @@ public String toString() {
         return sb.toString();
     }
 
+    @Override
+    public int getPhase() {
+        return Integer.MAX_VALUE;
+    }
+
+    @Override
+    public boolean isRunning() {
+        return isStarted();
+    }
+
+    @Override
+    public void start() {
+        try {
+            super.start();
+        } catch (Exception e) {
+            sneakyThrow(e);
+        }
+    }
+
+    @Override
+    public void stop() {
+        try {
+            super.stop();
+        } catch (Exception e) {
+            sneakyThrow(e);
+        }
+    }
+
+    @SuppressWarnings("unchecked")
+    private static <E extends Exception> void sneakyThrow(Exception e) throws E {
```

Comment:
Please use the existing API we have to wrap as runtime exception

    throw ObjectHelper.wrapRuntimeCamelException(e);